organisation-api-schema.yaml

```diff
@@ -33,57 +33,8 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Organisation_List'
-  '/organisations/{organisation_id}':
```

Comment:
Just double checking, I'm assuming you've taken it out as the conveyancer won't be able to hit this through gatekeeper?

Reply:
yes I removed all endpoints not exposed to conveyancers